sw/device/tests/pwrmgr_smoketest.c

```diff
@@ -106,6 +123,11 @@ bool test_main(void) {
 
   } else if (compare_wakeup_reasons(&wakeup_reason, &kWakeUpReasonTest)) {
     LOG_INFO("Aon timer wakeup detected");
+    LOG_INFO("Check the rstmgr reset_info is LOW_POWER timer wakeup detected");
+    dif_rstmgr_reset_info_bitfield_t info;
+    CHECK(dif_rstmgr_reset_info_get(&rstmgr, &info) == kDifRstmgrOk);
+    CHECK(info == (dif_rstmgr_reset_info_bitfield_t)(kDifRstmgrResetInfoPor |
+           kDifRstmgrResetInfoLowPowerExit));
```

Comment:
With Tim's confirmation, lets keep the clearing of `reset_info` during the POR phase so that here, we check `info` is exactly equal to `kDifRstmgrResetInfoLowPowerExit`.

Reply:
Yes, isolation here makes sense.